ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/services/ValueDeserialiser.java

```diff
@@ -0,0 +1,32 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.services;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+public class ValueDeserialiser extends StdDeserializer<List<String>> {
```

Comment:
I can't seem to find any usages for this class in your code, is it still needed?